packages/channel-client/tests/fakes/fake-channel-provider.ts

```diff
@@ -19,6 +19,8 @@ import {Wallet, utils} from 'ethers';
 
 const bigNumberify = utils.bigNumberify;
 
+type ChannelId = string;
```

Comment:
Is this the right place for this? Also, couldn't we use the wrapped type strategy such as what Andrew proposed / [newtype](https://github.com/gcanti/newtype-ts)?

Reply:
We could but that is out of scope for this PR. I could just revert to using string throughout but I thought it made it easier to see that the lookup tables stored in this class are all being keyed by a channel id.